themes/default/content/blog/pulumi-release-notes-m59/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "July 28 releases: K8s GitOps, autonaming in Google Native"
+date: 2021-07-28T08:00:00-07:00
+draft: false
+meta_desc: "Kubernetes is faster, K8s operator supports GitOps, new resources and functionality for Azure and Google Cloud, and easier getting started in the Pulumi Service"
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+Another 3 weeks have passed, so it's time for another edition of the Pulumi release notes! This iteration we've been busy smashing bugs across our products (over 100!), but we've still got some great new features to share with you as well:
```

Comment:
```suggestion
Another great iteration has just wrapped up, and we've got several new updates across Pulumi providers, CLI and service to share:
```

Reply:
Overall I like your re-word, but any reason you took out the bug smashing item? I think it's important to note to the community that we're fixing small things, not just always building new stuff.

Reply:
```suggestion
Another great iteration has just wrapped up, so it's time for another edition of the Pulumi release notes! We've been busy smashing bugs across our products (over 100!), but we've also got several new updates across Pulumi providers, the CLI, and the Pulumi Service:
```